modules/Link.js

```diff
@@ -73,6 +74,11 @@ const Link = React.createClass({
   },
 
   handleClick(event) {
+    invariant(
```

Comment:
We can't – the idea is that for unit tests, we let the link be rendered without a router, so that people can test non-navigation-related aspects of their page components that use `<Link>`s without having to set up a router context.